new-lamassu-admin/public/index.html

```diff
@@ -7,6 +7,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
+    <meta name="googlebot" content="noindex"/>
```

Comment:
Do we only want to block google, or do we want to block every search engine?
If we want to block every search engine, change "googlebot" to "robots"
(confirm with Rafael or whoever created the card)

Reply:
Card only mentioned google indexing, hence the googlebot. But you are right, it's better to generalize